install/build_from_source.md

```diff
@@ -56,7 +56,42 @@ To have ROOT setup automatically at each login, that command can be appended to
 
 ## ROOT, Python and PyROOT
```

Comment:
must add a disclaimer that _everything_ that follows is only applicable from ROOT v6.22 onwards